### PR TITLE
feat(about): Guide Native_install_version_14_and_upgrade_15

### PR DESCRIPTION
**Pull request by tuannn2**
Sw360 installation documents without docker version 14 and update to version 15

### CODE REVIEW

File: content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md [hunk 1]
```diff
@@ -0,0 +1,684 @@
...
+        backend.timeout.connection = 50000
+
+        ## Timeout for reading response from the backend
+        ## This defines the maximum wait time until the
+        ## first byte of the response is available.
+        backend.timeout.read = 6000000
+
+        #Log
+        enable.sw360.change.log=true
+        sw360changelog.config.file.location=/opt/sw360logs
```
Comment (KoukiHama) [inline content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md:641]: This is not default setting. 

#Uncomment the below file location if the log4j2.xml file is placed inside etc/sw360 folder.
#sw360changelog.config.file.location=/etc/sw360/log4j2.xml


https://github.com/eclipse/sw360/blob/63a1fa6211e97c39b84f60676804aa3cc83e0f2d/backend/src-common/src/main/resources/sw360.properties#L77-L80

File: content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md [hunk 2]
```diff
@@ -0,0 +1,684 @@
...
+        rest.write.access.usergroup=USER
+        project.obligations.enabled=true
+
+        ## Timeout for connecting to the backend
+        backend.timeout.connection = 50000
+
+        ## Timeout for reading response from the backend
+        ## This defines the maximum wait time until the
+        ## first byte of the response is available.
+        backend.timeout.read = 6000000
```
Comment (KoukiHama) [inline content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md:637]: This is not default setting, and need not mention here.

File: content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md [hunk 3]
```diff
@@ -0,0 +1,684 @@
...
+
+        rest.apitoken.generator.enable = true
+        #custom.welcome.page.guideline = true
+        sw360.liferay.company.id=20098
+
+        rest.write.access.usergroup=USER
+        project.obligations.enabled=true
+
+        ## Timeout for connecting to the backend
+        backend.timeout.connection = 50000
```
Comment (KoukiHama) [inline content/en/docs/Deployment/Upgrading/Native-Install-Version-14.0.0-and-Upgrade-Version-15.0.0.md:632]: This is not default setting, and need not mention here.

**Review by KoukiHama — CHANGES_REQUESTED** [review]
Basically,  ok
about the sw360.property,  please use these setting to be more general description, 
https://github.com/eclipse/sw360/blob/sw360-14.0.0-M1/backend/src-common/src/main/resources/sw360.properties

**Review by heliocastro — CHANGES_REQUESTED** [review]
It's ok, but i suggest two possible changes:
- Move to legacy, as is based on very old non supoprted anymore Ubuntu 18.04 OR
- Prepare the guide with Ubuntu 22.04

I'll be ok to merge like this and then move to 22.04

**Comment by KoukiHama** [comment]
and mark down should be split to" install 14 version page" and "upgrade from 14 to 15"
As Helio mentioned, "install 14 version" should be in https://github.com/eclipse/sw360.website/tree/master/content/en/docs/Deployment/Legacy

**Comment by tuannn2** [comment]
> It's ok, but i suggest two possible changes:
> 
> * Move to legacy, as is based on very old non supoprted anymore Ubuntu 18.04 OR
> * Prepare the guide with Ubuntu 22.04
> 
> I'll be ok to merge like this and then move to 22.04

**Comment by tuannn2** [comment]
Thank you for your review, @KoukiHama @heliocastro. I will change according to your comments!

**Comment by tuannn2** [comment]
hi @heliocastro, 

-  I have updated the 14 installation md file, from ubuntu 18 to 20. Please check it!
-  I am preparing and testing SW360 on ubuntu 22.04.

**Comment by tuannn2** [comment]
hi @KoukiHama.  I split the md file into 2 install 14 and updated to 15. Please check it!

**Review by KoukiHama — APPROVED** [review]

**Comment by KoukiHama** [comment]
@tuannn2  Thank you! It is OK for me now.

**Pull request merged**